customer-satisfaction/README.md

```diff
@@ -24,13 +24,32 @@ If you are running the `run_deployment.py` script, you will also need to install
 
 ```bash
 zenml integration install mlflow -f
+``` 
+The zenfile can only be executed with a ZenML stack that has an MLflow model deployer as a component. Configuring a new stack with a MLflow model deployer could look like this:
+```bash
+zenml integration install mlflow -f
+zenml model-deployer register mlflow --type=mlflow
+zenml stack register local_with_mlflow -m default -a default -o default -d mlflow
+zenml stack set local_with_mlflow
 ```
 
-## :thumbsup: The Task
+## 📙 Resources & References
+
+We had written a blog that explains this project in-depth: [Predicting how a customer will feel about a product before they even ordered it](https://blog.zenml.io/customer_satisfaction/).
+
+If you'd like to watch the video that explains the project, you can watch the [video](https://youtu.be/L3_pFTlF9EQ).
```

Comment:
can you embed here as well?

Reply:
We can embed once the videos and blogs are ready :)